packages/wp-now/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import os from 'os';
+import path from 'path';
+
+export const WP_NOW_PATH = path.join(os.homedir(), '.wp-now');
+export const WORDPRESS_ZIPS_PATH = path.join(WP_NOW_PATH, 'wordpress_zips');
+export const SQLITE_ZIP_PATH = path.join(WP_NOW_PATH, 'sqlite_zip');
+export const SQLITE_FILENAME = 'sqlite-database-integration';
+export const SQLITE_PATH = path.join(SQLITE_ZIP_PATH, `${SQLITE_FILENAME}-main`);
+export const WP_DOWNLOAD_URL = 'https://wordpress.org/latest.zip'
+export const SQLITE_URL = 'https://github.com/WordPress/sqlite-database-integration/archive/refs/heads/main.zip'
+export const DEFAULT_PORT = 8881;
```

Comment:
it would be really nice if we had descriptive docblock comments for each of these. for instance, I'm sure `WP_NOW_PATH` makes way more sense when writing this code than when reading it in some other spot that imports it.

would be nice if when hovering over `WP_NOW_PATH` we had some background on what it means and why it exists. same for the others

Reply:
ChatGPT could be helpful with the first draft

Reply:
I added the descriptions here https://github.com/WordPress/wordpress-playground/pull/188/commits/a845a6043395c3fe6ad81a5bb76899dd2db059c5